packages/itwinui-react/src/core/Buttons/SplitButton/SplitButton.tsx

```diff
@@ -28,6 +28,14 @@ export type SplitButtonProps = ButtonProps & {
    * Content of primary button.
    */
   children: React.ReactNode;
+  /**
+   * Passes props to SplitButton wrapper.
+   */
+  wrapperProps?: React.ComponentProps<typeof Box>;
```

Comment:
should just use div props

```suggestion
  wrapperProps?: React.ComponentProps<'div'>;
```